docs/_workshops/object-detection/Annotations to Colab.md

```diff
@@ -0,0 +1,27 @@
+
+---
+title: Training a Model
+order: 6
+redirect_from:
+  - /object-detection/cli/6.html
+---
+After we have collected and labeled our first round of images, we are ready to start training our model! 
+
+## Prepare to Export to Colab
+To train our model we need to install the Cloud Annotation CLI.
+
+#### Click Train in Colab
+There's a green button on the top right of your screen that says "Train in Colab". Click that to get started.
```

Comment:
label this section something like `Training with Google Colab` and add a short summary of what colab is and why we are using it.

Also it feels weird to specify the color of the button

Reply:
also I normally use back ticks when specifying UI text to click on

Reply:
I made that change.